components/tabs/tabs.js

```diff
@@ -164,7 +164,6 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(RtlMixin(FocusVisiblePolyf
 				background-color: transparent;
 				border: 1px solid transparent;
 				border-radius: 15px;
-				box-shadow: 0 0 0 4px rgba(0, 0, 0, 0);
```

Comment:
It looks as though this wasn't doing anything. I suspect this was left-over from some previous implementation/work and never got cleaned up.